### PR TITLE
Clarify persistent storage settings.

### PR DESCRIPTION
**Pull request by seanfcarroll**
Please confirm this is correct.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,12 +179,13 @@ Elasticsearch](https://github.com/deviantony/docker-elk/wiki/Elasticsearch-clust
 The data stored in Elasticsearch will be persisted after container reboot but not after container removal.
 
 In order to persist Elasticsearch data even after removing the Elasticsearch container, you'll have to mount a volume on
-your Docker host. Update the `elasticsearch` service declaration to:
+your Docker host. Edit ```docker-compose.yml``` and add the path to persistent storage underneath the existing entry in the `elasticsearch` service declaration:
```
Comment (antoineco) [inline README.md]: @seanfcarroll thanks for the suggestion! Your addition is correct but maybe a bit verbose. How about something more succinct like:

```suggestion
your Docker host. Add the following bind mount to the existing volumes of the `elasticsearch` service declaration:
```

File: README.md [hunk 2]
```diff
@@ -179,12 +179,13 @@ Elasticsearch](https://github.com/deviantony/docker-elk/wiki/Elasticsearch-clust
...
 
 ```yml
 elasticsearch:
 
   volumes:
+    - ./elasticsearch/config/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml:ro
```
Comment (antoineco) [inline README.md]:
```suggestion
    # replace <path to storage> with the path of the host directory to be mounted
```

File: README.md [hunk 3]
```diff
@@ -179,12 +179,13 @@ Elasticsearch](https://github.com/deviantony/docker-elk/wiki/Elasticsearch-clust
...
     - /path/to/storage:/usr/share/elasticsearch/data
```
Comment (antoineco) [inline README.md]:
```suggestion
    - <path to storage>:/usr/share/elasticsearch/data
```

File: README.md (hunk 1 above)
Comment (antoineco) [reply README.md]: Ping @seanfcarroll

File: README.md (hunk 3 above)
Comment (DerekChia) [reply README.md]: Perhaps to make it clearer for the uninitiated, you can use the phrase `local storage` instead of just `storage`. Thanks.

**Comment by antoineco** [comment]
Obsoleted by #430

**Pull request closed without merging**